### PR TITLE
Id for each link to make individual styles possible

### PR DESCRIPTION
**Pull request by paeddl**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Adds an ID to each link at "TermsAndConditions" in checkout |
| Type? | improvement |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
ping @vincentbz

**Comment by superGB** [comment]
status: PM approved
This feature is required for the certification with EHI.

**Comment by mickaelandrieu** [comment]
@paeddl would you mind to fix the unit tests please ?

**Comment by paeddl** [comment]
@mickaelandrieu 
just done: https://github.com/PrestaShop/PrestaShop/commit/764aedead700a1287daf2d850166edf9b5c1afd2

File: tests/Unit/Core/Business/Checkout/TermsAndConditionsTest.php [hunk 1]
```diff
@@ -16,15 +16,15 @@ public function setup()
     public function test_SetText_InsertsLinks()
     {
         $this->assertEquals(
-            'hello <a href="http://www.world.com">world</a>',
+            'hello <a href="http://www.world.com" id="cta--0">world</a>',
```
Comment (mickaelandrieu) [inline tests/Unit/Core/Business/Checkout/TermsAndConditionsTest.php]: Dont you expect an identifier to be inserted?

Comment (paeddl) [inline tests/Unit/Core/Business/Checkout/TermsAndConditionsTest.php]: @mickaelandrieu 
you are right... I adapted the tests again: 
https://github.com/PrestaShop/PrestaShop/commit/7abffa419b4d3b8f7381308e3cbe7bb03b01fec7

**Comment by mickaelandrieu** [comment]
@paeddl Thank you, good job!

**Pull request merged**